README.md

```diff
@@ -25,3 +25,9 @@ The folling plugins are included with this repository:
 
 * [Key Vault](./src/Microsoft.Azure.ServiceBus.KeyVaultPlugin/readme.md)
 * [Message ID](./src/Microsoft.Azure.ServiceBus.MessageIdPlugin/readme.md)
+
+## Third party provided plugins
+
+* [Claim Check pattern](https://www.nuget.org/packages/ServiceBus.AttachmentPlugin)

```

Comment:
Also, curious. Do you want to link it to your GitHub repo instead of nuget package?